README.md

```diff
@@ -105,7 +105,8 @@ Agents facilitate the interaction from clients with canisters on the Internet Co
 ### Dart/Flutter
 
 - [agent_dart](https://github.com/AstroxNetwork/agent_dart) - Framework to build mobile applications.
-- [ic_dart_tools](https://github.com/levifeldman/ic_tools_dart) - Tools supporting Flutter on the web.
+- [ic_tools](https://github.com/levifeldman/ic_tools) - IC Tools supporting Dart & Flutter on the web and linux.
+- [ic_tools_web](https://github.com/levifeldman/ic_tools_web) - Library with specific tools for Flutter on the Web.  
```

Comment:
Thank you @levifeldman for the contribution. Little nitpick: Could you use consistent casing for `web`?

Reply:
Sure